README.md

```diff
@@ -143,6 +143,25 @@ You'll need apache2-utils on the machine where you plan to create the htpasswd f
 
 If you need to configure Nginx beyond what is possible using environment variables, you can provide custom configuration files on either a proxy-wide or per-`VIRTUAL_HOST` basis.
 
+#### Overriding default proxy settings
+
+If you want to override the default proxy settings for the nginx container, add a configuration file at `/etc/nginx/proxy_config.conf`. A file with the default settings would
```

Comment:
`_config` is still redundant and superfluous.  I'd rather use `proxy.conf` since it's shorter and matches up with the nginx examples: http://wiki.nginx.org/FullExample